README.md

```diff
@@ -18,6 +19,11 @@ Open [http://localhost:8080](http://localhost:8080) to view it in the browser.
 The page will reload if you make edits.\
 You will also see any lint errors in the console.
 
+Note on local authentication - you will have to grab your `token` from https://stream.resonate.coop.
+Log in normally through the web app and then visit your profile's endpoint to view your `token`:
```

Comment:
```suggestion
Log in normally through the [web app](https://stream.resonate.coop) and then visit your profile's endpoint to view your `token`:
```

Adding a link here just to be extra thorough. People might get lost between all the different "web player" "stream app" and offerings in our current ecosystem so being extra explicit here might help out newer folks.